capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -13,9 +13,40 @@ function createClassroom() {
     var classroomData = JSON.stringify({ "nickname": name, "userId": email, "subject": subject });
     fetch("/dashboard-handler", {method: "POST", body: classroomData}).then((resp) => {
         if (resp.ok){
-            console.log("Classroom Created!");
+            getClassrooms();
         } else {
             console.log("Error has occured");
         }
     });
 }
+
+function getClassrooms() {
+    fetch("/dashboard-handler").then(response => response.json()).then((classroomsList) => {
+        const classroomElement = document.getElementById("classroom-container");
+        classroomElement.innerHTML = "";
+        for (classroom of classroomsList) {
+            classroomElement.appendChild(createClassroomDivElement(classroom));
+        };
+    });
+}
+
+function createClassroomDivElement(classroom) {
```

Comment:
Recommend using string templates and either DOMParser or innerHTML.

https://developer.mozilla.org/en-US/docs/Web/API/DOMParser

Reply:
Alternatively, this would be a place handlebars.js could be useful